frontend/src/scenes/onboarding/sdks/SDKs.tsx

```diff
@@ -41,7 +58,29 @@ export function SDKs({
         width && setShowSideBySide(width > minimumSideBySideSize)
     }, [width])
 
-    return (
+    return !isFirstProduct && hasSnippetEvents === null ? (
```

Comment:
Nit but I assume that the value for hasSnippetEvents would be cached so I wouldn't see the flash between this first branch and the "huzzah!" branch each time I toggle back to the ?sdks page from a product intro page.